### DISCUSSION TITLE
Running a normal-tumor pair on "multi" mode

### DISCUSSION THREAD

**Discussion by smahesh12** [uclahs-cds/metapipeline-DNA#112] [General]
Would it be feasible to run a group of patients (some of which have 1N-1T, some of which have 1N-2T) while specifying "multi" in the config file? Obviously, I could split them into two input CSV files with two configs (one with "paired", and one with "multi"). But is there a way to run all these in the same input CSV with the same config ("multi")?

@tyamaguchi-ucla @yashpatel6 
Thanks!

**Comment by yashpatel6** [comment]
You should be able to use the same config and use `multi`; the metapipeline should handle both paired and multi with that setting.

**Reply by smahesh12** [reply]
Perfect, thank you!
